tutorials/AMB2018-01-L7/system/decomposeParDict

```diff
@@ -14,7 +14,7 @@ FoamFile
 }
 // * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
 
-numberOfSubdomains 4;
+numberOfSubdomains 16;
```

Comment:
do we want to set this lower so people that download can still run on their laptops? 9 would probably work for most people these days and is still a perfect square

Reply:
Yes - reverted